src/Kernel/client/telemetry.ts

```diff
@@ -34,94 +36,97 @@ class LiteEvent<T> implements ILiteEvent<T> {
     }
 }
 
-
-
 export interface ClientInfo {
     Id: string;
     IsNew: boolean;
     CountryCode: string;
-    CookieConsentMarkup: ConsentMarkup;
     HasConsent: boolean;
     FirstOrigin: string;
 }
 
-interface ConsentMarkup {
-    Markup: string;
-    Javascripts: string[];
-    Stylesheets: string[];
+enum consentCategories {
```

Comment:
Done.

Reply:
I reverted it to enum because I really want to use it as an enum in 
this call `this.siteConsent.getConsentFor(consentCategories.Required);` instead of a string literal.